app/views/moderations/mod.html.erb

```diff
@@ -96,6 +104,16 @@
       This is the equivalent of vomiting on <b>all</b> of this user's articles.
     </p>
   </div>
+  <% if (current_user.any_admin? || current_user.tech_admin?) && @moderatable.class.name == "Comment" && !@moderatable.deleted %>
```

Comment:
Let's make sure this is only available to `current_user.any_admin?` and not tech admins.